### PR TITLE
Add stale and last updated at info to cards. Refactor a lot

### PR DESCRIPTION
**Pull request by Utwo**
### What does it fix?
Add stale and last updated at info to cards.
Add counties info back.

### Refactors
A lot of this changes was taken from ce-ma-fac repo.
Load font from local
Add google analytics
Update fe-components
Move images in src
Remove bottom-nav and top-nav and use the components from fe-components
Remove footer components and use the footer component from fe-components
Remove date-fns and use native Date

### CODE REVIEW

File: frontend/src/components/cards/age/age.jsx [hunk 1]
```diff
@@ -64,21 +65,21 @@ export class AgeCard extends React.PureComponent {
...
       ]
-    }
-  }
+    };
+  };
 
   render() {
     const { title, state } = this.props;
-    const { isLoaded, error } = state;
+    const { isLoaded, error, lastUpdatedOnString, stale } = state;
```
Comment (moonflare) [inline frontend/src/components/cards/age/age.jsx:79]: nit
```suggestion
    const { isLoaded, error, lastUpdatedDate, stale } = state;
```

File: frontend/src/components/cards/counties-map/counties-map.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          type: 'map',
+          mapType: 'RO',
+          itemStyle: {
+            areaColor: Constants.curedColor
+          },
+          label: {
+            normal: {
+              show: false
+            }
+          },
```
Comment (tudoramariei) [inline frontend/src/components/cards/counties-map/counties-map.jsx:46]: Replacing this section with the following code will fix #286 by hiding all the county names from the map
```javascript
emphasis: {
    label: {
        show: false
    }
},
```

File: frontend/src/components/cards/counties-map/counties-map.jsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      };
+    });
+    return {
+      tooltip: {
+        trigger: 'item'
+      },
+      visualMap: {
+        show: false,
+        min: 0,
+        //max: this.props.state.max,
```
Comment (moonflare) [inline frontend/src/components/cards/counties-map/counties-map.jsx:24]:
```suggestion
```

File: frontend/src/components/cards/counties-table/counties-table.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { mnemonics } from '../../../config/mnemonics';
+import { Card } from '../../layout/card/card';
+
+export const EMBED_COUNTIES_TABLE = 'counties-table';
+
+export class CountiesTable extends React.PureComponent {
+  render() {
+    const state = this.props.state;
```
Comment (moonflare) [inline frontend/src/components/cards/counties-table/counties-table.jsx]: nit
```suggestion
    const { state } = this.props;
```

File: frontend/src/components/layout/card/card.jsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        <div className="is-error is-block">Nu am putut încărca datele</div>
+      );
+    }
+
+    return isLoaded ? (
+      <div className={`card ${isStale ? 'card--warning' : ''}`}>
+        {title && (
+          <header className="card-header">
+            <div className="card-header-title has-text-weight-normal">
+              <h3 className="card-title has-text-weight-bold">{title}</h3>
```
Comment (moonflare) [inline frontend/src/components/layout/card/card.jsx:55]:
```suggestion
              {title && <h3 className="card-title has-text-weight-bold">{title}</h3>}
```

File: frontend/src/components/cards/counties-map/counties-map.jsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+            }
+          },
+          data
+        }
+      ]
+    };
+  }
+
+  render() {
+    const state = this.props.state;
```
Comment (moonflare) [inline frontend/src/components/cards/counties-map/counties-map.jsx:54]: nit
```suggestion
    const { state } = this.props;
```

File: frontend/src/components/cards/age/age.jsx [hunk 7]
```diff
@@ -36,16 +37,13 @@ export class AgeCard extends React.PureComponent {
         axisPointer: {
           axis: 'y'
         },
-        formatter: (rawData) => {
-          const item = rawData[0]
-          return `${item.name}: ${item.data.value} (${item.data.percentage}%)`
-        },
+        formatter: rawData => {
+          const item = rawData[0];
```
Comment (moonflare) [inline frontend/src/components/cards/age/age.jsx:41]: nit
```suggestion
          const [ item ] = rawData;
```

File: frontend/src/components/cards/age/age.jsx (hunk 1 above)
Comment (Utwo) [reply frontend/src/components/cards/age/age.jsx:79]: In dashboard we send lastUpdatedOnString

Comment (moonflare) [reply frontend/src/components/cards/age/age.jsx:79]: Ah right. We could alias, but works as it is 👍

**Comment by danichim** [comment]
Closes #280

**Comment by Utwo** [comment]
Closes #279

**Comment by tudoramariei** [comment]
Closed both issues. Thanks, @danichim & @Utwo

**Pull request merged**